release-notes/opendistro-for-elasticsearch-anomaly-detection-kibana-plugin.release-notes-1.10.1.0.md

```diff
@@ -0,0 +1,52 @@
+## Version 1.10.1.0 Release Notes
+
+Compatible with Kibana 7.9.1
+
+### Features
+
+* Add sample detectors and indices (#272)
```

Comment:
just curious: previous release note has link associated with each PR, like this: https://raw.githubusercontent.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/master/release-notes/opendistro-for-elasticsearch.anomaly-detection-kibana-plugin.release-notes-1.9.0.0.md

Will simply putting #<PR number> in the release notes file point user to actual link of PR?

Reply:
Oh whoops, yeah, the links need to be added. @saratvemulapalli can you run the helper script to generate the links?

Reply:
Thats a good point. Sure. Let me take care of that.